integrations/whatsapp/integration.definition.ts

```diff
@@ -3,6 +3,36 @@ import { sentry as sentryHelpers } from '@botpress/sdk-addons'
 import { z } from 'zod'
 
 export const name = 'whatsapp'
+export const channel = 'channel' // TODO: Rename to "whatsapp" once support for integration versioning is finished.
```

Comment:
**question:** why would you want to rename the channel ? I'm curious. I know `"channel"` is redundant, but `"whatsapp"` doesnt seem much better and it would make this integration different for no reason. Let's talk about it before actually doing the change

Reply:
This is because people have complained on Discord that they tried to use `event.channel` in their bot to know which channel is the bot currently running on for a given message/conversation, but it always returns the useless value "channel" 😆